tests/unittest_nodes.py

```diff
@@ -1583,6 +1583,15 @@ def func():
     """
     )
     assert node.doc == "Docstring"
+    assert isinstance(node.doc_node, nodes.Const)
```

Comment:
Can you add a multi-line docstring to test their behaviour for `FunctionDef` nodes?

Reply:
This is already tested in test_with_multiline_docstring for FunctionDef